docs/src/main/tut/service.md

```diff
@@ -129,35 +129,49 @@ A builder can be `run` to start the server.
 val server = builder.run
 ```
 
+Go ahead and use curl (or your favorite HTTP client) to see your service in
```

Comment:
Fine as-is.  Could also be "Use curl, or your favorite HTTP client, to see your service in action:".